skill-tree /src/main/java/com/RDS/skilltree/User/UserController.java

```diff
@@ -0,0 +1,4 @@
+package com.RDS.skilltree.User;
+
+public class UserController {
```

Comment:
Any use for this class?

Reply:
This is there in different PR, we can remove this @bhtibrewal